### ISSUE TITLE
problem running spearmint beyond the provided examples

### ISSUE THREAD

**Issue by kirk86** [JasperSnoek/spearmint#61]
Hi everyone,
apologies for the disturbance. I'm having some problem running spearmint to optimize the learning rate in a neural net.

What I've done is the following:

`1. created a new folder with a wrapper function and the config.pb inside`
`2. for the wrapper function I've renamed  the file in which I had the actual network implementation into training.py`
`3. inside the training.py there are a number of different functions. I've also added another function named training which is the main loop.`
`4. in the main loop of the neural net I also call the objective function which updates the parameters and returns the learning rate.`
`5. I assign the returned learning rate to a new variable named results as in the examples of branin.py`
`6. I've created the main function also as in the examples of the branin.py. Actually it's just a copy paste with a few changes.`
`7. in my config.pb I've set up only one variable the learning_rate to be a float of size 1 with min=-5 and max=5.`

When I run the code the outputs that I get are all zero. I don't see any best value, mean value, number of tries, etc... Its like the GP has some sort of problem sampling the first few points.

Thanks!
